cjworkbench/i18n/catalogs/__init__.py

```diff
@@ -0,0 +1,11 @@
+from os import path
+
+CATALOG_FILENAME = "messages.po"
+BACKUP_CATALOG_FILENAME = "old.po"
```

Comment:
I explain it in the merging script: it's a temp file created for each locale before lingui's `extract` and used after lingui's `extract`. Since the name is used in two places (the one that writes it and the one that reads it), I made it a constant.

Reply:
Ah ... k, two things:

1. I think "backup" is the wrong word: backup suggests something that will not be deleted. This is a tempfile, not a backup file. Could you please choose a name based on its purpose?
2. Don't put the constant in `catalogs/__init__.py`. It should be private to `merge.py` because only `merge.py` will ever use it.

Reply:
Okay, I see the distinction. Now let's make the API distinguish them :).

How about: `i18n.catalogs.load_catalog(locale_id: str) -> babel.messages.catalog.Catalog`?

(For bonus points, loading the Catalog in the global function would be extra-great because we could use an `@lru_cache()` so we'd only need to load each catalog once.)